concent_api/signing_service_integration_test_recovery.py

```diff
@@ -0,0 +1,242 @@
+#!/usr/bin/env python3
+
+from configparser import ConfigParser
+from time import sleep
+from typing import Any
+import os
+import socket
+import subprocess
+import sys
+
+from api_testing_common import assert_condition
+from api_testing_common import count_fails
+from common.helpers import get_current_utc_timestamp
+from middleman_protocol.message import GolemMessageFrame
+from middleman_protocol.concent_golem_messages.message import SignedTransaction
+from middleman_protocol.stream import send_over_stream
+from middleman_protocol.stream import unescape_stream
+from signing_service_testing_common import Components
+from signing_service_testing_common import create_golem_message_frame_with_correct_transaction_signing_request
+from signing_service_testing_common import run_tests
+from signing_service_testing_common import send_message_to_middleman_and_receive_response
+from signing_service_testing_common import send_message_to_middleman_without_response
+
+
+REQUIRED_COMPONENTS = []  # type: ignore
+
+SLEEP_TIME_AFTER_SPAWNING_OR_KILLING_PROCESS = 10
```

Comment:
Please have two separate constants - one for SPAWNING_WAIT_TIME and one for KILLING_WAIT_TIME. Right now they are equal, but this might change and they are conceptually different, so we should have them separated.

Reply:
:+1: done